src/hb-ot-shape-complex-use-table.cc

```diff
@@ -194,7 +194,24 @@ static const USE_TABLE_ELEMENT_TYPE use_table[] = {
   /* 0DE0 */     O,     O,     O,     O,     O,     O,     B,     B,     B,     B,     B,     B,     B,     B,     B,     B,
   /* 0DF0 */     O,     O,  VPst,  VPst,     O,     O,     O,     O,
 
-#define use_offset_0x1000u 1360
+#define use_offset_0x0f18u 1360
+
+
+  /* Tibetan */
+                                                                     VMBlw, VMBlw,     O,     O,     O,     O,     O,     O,
```

Comment:
@ebraminio where do these values come from? I can't find them in the MS spec

Reply:
This is the result of applying of the change above on "gen-use-table.py" and re-running it. According to Behdad every one of them should be decided to what category they fall into and are hard to decide.

Reply:
I see, thanks a lot!